packages/esbuild/esbuild.bzl

```diff
@@ -233,6 +236,13 @@ and cjs when platform is node. If performing code splitting or multiple entry_po
 See https://esbuild.github.io/api/#format for more details
         """,
         ),
+        "include_ts_declarations": attr.bool(
```

Comment:
can we think of a use case where you'd need to set this to True? I'd rather not introduce the user-facing complexity of more configuration affordances if they're likely not useful and just out of FUD

Reply:
One use case I've though of is if you want your JS to bundle to invoke TSC. For example, at this point I've made my one target filter out tsc outputs and not invoke the compiler but I've made a separate target invoke not filter those outputs out so that I can run that if I want the compiler to run during my dev cycle

Reply:
I don't understand still. If your workflow is "I want to ensure my code type-checks" then why would you `bazel build :some-esbuild-target` to do that? The esbuild target still does nothing with the extra inputs.

Reply:
Done